src/commands/cmd_key.cc

```diff
@@ -64,6 +64,41 @@ class CommandMove : public Commander {
   }
 };
 
+class CommandMoveX : public Commander {
+ public:
+  Status Execute(Server *srv, Connection *conn, std::string *output) override {
+    std::string &key = args_[1], &ns = args_[2], &token = args_[3];
+
+    redis::Database redis(srv->storage, conn->GetNamespace());
```

Comment:
Is it better to move the old AuthenticateUser as a member function and set the connection namespace outside? so that we can reuse the authenticate code block:

```
AuthResult Server::AuthenticateUser(const std::string &user_password, std::string *ns);
```